test/uk/gov/hmrc/alcoholdutycalculator/services/DutyServiceSpec.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.alcoholdutycalculator.services
+
+import uk.gov.hmrc.alcoholdutycalculator.base.SpecBase
+import uk.gov.hmrc.alcoholdutycalculator.models.{AlcoholByVolume, DutyCalculationRequest, Volume}
+
+class DutyServiceSpec extends SpecBase {
+  val dutyService = new DutyService()
+
+  "dutyService" should {
+    "calculate pure alcohol volume and duty" in {
```

Comment:
Can you ad a few more test cases please? Like calculating values without decimals, negative values and a few more rounding cases